feature/_fish/install.sh

```diff
@@ -2,10 +2,14 @@
 
 if [[ "${FLAVOUR}" =~ (simple|tanzu|gcloud) ]]
 then
-  execHandle 'Installing fish' sudo apk add fish perl fzf git
+  execHandle 'Installing fish' sudo apk add fish fzf git perl
 elif [[ "${FLAVOUR}" =~ (aws|azure) ]]
 then
   prepare
+  if [[ "${FLAVOUR}" == 'azure' ]]
+  then
+    execHandle 'Installing aws' sudo tdnf install -y gawk
```

Comment:
```suggestion
    execHandle 'Installing awk' sudo tdnf install -y gawk
```

I guess, that's a typo here

Reply:
Yes, it obviously didn't really help to read _aws_ a few times in the regexp above... 🤦